Admin/Field/Action.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace LAG\AdminBundle\Admin\Field;
+
+class Action extends Link
```

Comment:
No, an Field\Action is a Field, it display link to Admin\Action, while a Link field will display Entity data into a link form.